src/chains/filecoin/filecoin/src/deal-state.ts

```diff
@@ -1,46 +1,127 @@
 // Note that the runtime number values of these enums match
-// *exactly* the number values of the same states on the Lotus server.
+// *exactly* the number values of the same states found at
+// go-fil-markets/storagemarket/dealstatus.go (https://git.io/JtJAS)
+// **DO NOT TRUST** other sources (i.e. lotus/lotuspond/front/src/Client.js
+// or js-lotus-client-workshop/src/08-deals/deal-list.js)
 // Don't reorganize unless you know what you're doing.
 
+// Updated to v1.1.1 (https://github.com/filecoin-project/go-fil-markets/blob/v1.1.1/storagemarket/dealstatus.go)
 export enum DealState {
+  // Unknown means the current status of a deal is undefined
```

Comment:
If you do keep these as an enum instead of a map (see above comment), you can use JSDoc-style comments to have intellisense pick up these descriptions for us. e.g.,

![image](https://user-images.githubusercontent.com/187813/104788668-ce076480-5760-11eb-9b5a-a45c700c5851.png)

instead of:

![image](https://user-images.githubusercontent.com/187813/104788709-e7a8ac00-5760-11eb-9a51-aa6f7fc341e3.png)

Reply:
Would you prefer that we have the JSDoc, requiring further manipulation from copying this from the source or  leave it as is to be as close/consistent with the golang source?